fnf/README.md

```diff
@@ -0,0 +1,15 @@
+## Microsoft Academic Knowledge API ##
```

Comment:
Link to the main documentation https://docs.microsoft.com/en-us/azure/cognitive-services/academic-knowledge/home too